{"uuid": "1b5daa96-2b1e-478b-8d67-9d4b260f1172", "children": ["9114c996-6653-4c0b-8e4c-5ab0b51dd9b4"], "befores": [{"name": "set_up_org", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='test.hkciot.com', port=443): Max retries exceeded with url: /cuteview/org/get?cropId=1751805517940535298 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002707FF03520>, 'Connection to test.hkciot.com timed out. (connect timeout=None)'))\n", "trace": "  File \"D:\\sw_install\\python-install\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\_pytest\\fixtures.py\", line 909, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\ui\\test_case\\test_terminal_center.py\", line 38, in set_up_org\n    org_list.clear_all_org()\n  File \"D:\\ui\\common\\set_up_org.py\", line 48, in clear_all_org\n    ids = self.get_org_ids()\n  File \"D:\\ui\\common\\set_up_org.py\", line 22, in get_org_ids\n    reponse = requests.get(url=base_url, headers=self.headers, params=body)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\api.py\", line 73, in get\n    return request(\"get\", url, params=params, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"D:\\sw_install\\python-install\\lib\\site-packages\\requests\\adapters.py\", line 553, in send\n    raise ConnectTimeout(e, request=request)\n"}, "start": 1715759204984, "stop": 1715759226032}], "start": 1715759204984, "stop": 1715759226083}